data/styles/default/include/Icons.mapcss

```diff
@@ -416,6 +416,10 @@ node|z17-[aerialway=station],
 node|z17-[railway=level_crossing]
 {icon-image: railway-crossing-m.svg;}
 
+node|z12-13[railway=station][station=funicular],
+{icon-image: funicular-s.svg;icon-min-distance: 10;}
```

Comment:
nit: `icon-min-distance: 10;` is not really needed and its use is controversial, we're likely to get rid of it

So you just one `z12-14` rule is enough.